docs/marketplace/user/features/202108.0/marketplace-product-feature-overview.md

```diff
@@ -37,6 +37,24 @@ The marketplace product is also displayed with the **Sold By** field defining th
 
 ![marketplace product on PDP](https://spryker.s3.eu-central-1.amazonaws.com/docs/Marketplace/user+guides/Features/Marketplace+product/add-merchant-product-to-wl-and-from-wh-to-cart.gif)
```

Comment:
the gif should be updated to b2b marketplace demo shop

Reply:
@ Marketplace Wishlist is a b2b feature, so we can't update it to b2b

Reply:
this is a shopping list document not wishlist. and the gif is taking from an outdated demo shop

Reply:
@AdvaBD ok, so I'll remove the outdated git, add a gif for a shopping list, right?

Reply:
the shopping list here is only in the context of quick order, it needs to show quick order in the different yves pages

Reply:
Got you, so the give must show Quick Order from all pages:
- Quick Order page
- Quick Add to Cart (on the Cart page)
- Quick Add to Shopping List (on Shopping List View and Edit pages)

Reply:
yes :-)